WowCharComparerWebApp/Controllers/User/RegisterController.cs

```diff
@@ -102,28 +101,26 @@ public bool CheckEmail(string email, ComparerDatabaseContext db)
 
         public void SendVerificationMail(string userEmail, Guid activationGuid)
         {
-            var test = HttpContext.Request.Path;
-
             SmtpClient client = new SmtpClient("smtp.gmail.com", 587)
             {
                 DeliveryMethod = SmtpDeliveryMethod.Network,
                 UseDefaultCredentials = false,
                 Credentials = new NetworkCredential(APIConf.WowCharacterComparerEmail, APIConf.WoWCharacterComparerEmailPassword),
                 Timeout = 100000,
-                EnableSsl = true             
+                EnableSsl = true
             };
 
             string protocol = HttpContext.Request.IsHttps ? "https" : "http";
 
             string activationLink = $"{protocol}://www.{ HttpContext.Request.Host}/Register/AccountConfirmation";
 
-            MailMessage msg = new MailMessage();    
+            MailMessage msg = new MailMessage();
             msg.To.Add(userEmail);
             msg.From = new MailAddress(APIConf.WowCharacterComparerEmail);
             msg.IsBodyHtml = true;
-            msg.Subject = "Verification";          
+            msg.Subject = "Welcome! Remember to verify your account.";
```

Comment:
I think it should be more like: 
"Battle.net Developer Portal Account Confirmation"  (subject from email sent by battlenet platform)